### PR TITLE
Use host and path to construct url for Segment.

### PR DESCRIPTION
**Pull request by brianhw**
When a 'page' is not already available.

### CODE REVIEW

File: eventtracking/backends/segment.py [hunk 1]
```diff
@@ -69,6 +74,13 @@ def send(self, event):
         path = context.get('path')
         referer = context.get('referer')
         page = context.get('page')
+
+        if path and not page:
+            # Try to put together a url from host and path:
+            host = context.get('host')
+            if host:
+                page = urljoin("//{host}".format(host=host), path)
+
```
Comment (mulby) [inline eventtracking/backends/segment.py]: Is there already test coverage for this edge case somewhere?

Comment (brianhw) [reply eventtracking/backends/segment.py]: This is why we have code reviews.   Added a test.
